src/assembler/preprocessor.rs

```diff
@@ -333,7 +333,11 @@ fn handle_symbol<'a, P: TokenProvider<'a>>(
     iter.consume_until(position); // includes the item
```

Comment:
Does this ever do anything? Isn't position behind the brace?

Reply:
Well, I'm not sure why I have this here, but...

I tend to leave per-line comments after I've fixed something super weird. This line might be in response to a bug (that I have since completely forgotten). The comment I left doesn't really explain it, so if deleting this line experimentally doesn't change anything, then we're probably good.